aws-iam-role-poweruser/main.tf

```diff
@@ -85,6 +85,7 @@ data "aws_iam_policy_document" "misc" {
       "iam:DeleteUser",
       "iam:CreateGroup",
       "iam:DeleteGroup",
+      "route53domains:RegisterDomain",
```

Comment:
Would more generally denying `route53domains:*` be better?

Reply:
I guess it doesn't really matter if we don't have domains to start with.

Reply:
I thought about that - but I think that the intention here was just to remove the ability of power users to register domains (because we want to force them to use MarkMonitor for that) and there's no real harm that I see in keeping the other route53domains permissions there. @austinylin do you have any thoughts?